user-journey-tests/specs/receiving-the-licence/postExit.spec.js

```diff
@@ -0,0 +1,15 @@
+import postExistPage from '../../page-objects/receiving-the-licence/postExistPage.js'
+
+describe('Post exit page test', () => {
+  beforeEach('Navigate to exit page', async () => {
+    await postExistPage.navigateToPageAndVerifyTitle()
+  })
+
+  it('Should verify the complete application link', async () => {
+    await postExistPage.verifyCompleteApplicationLink()
+  })
+
+  it('Should verify the continue with email button', async () => {
+    await postExistPage.verifyContinueWithEmailButton()
+  })
```

Comment:
Perhaps we could add a test here to check that if you later go back, the radio button for receive method has been set to "email"